haas/cli.py

```diff
@@ -295,7 +295,9 @@ def serve(port):
     # We need to import api here so that the functions within it get registered
     # (via `rest_call`), though we don't use it directly:
     from haas import model, api, rest
-    server.init(stop_consoles=True)
+    server.init()
+    server.check_db_schema()
```

Comment:
haas-admin db create calls server.init(), but it can't call check_db_schema, since that would cause a circular dependency; it's job is to initialize the db, but that would cause it to bomb out every time.